db/migrations/000006_hyperlane_transfers.up.sql

```diff
@@ -12,5 +12,8 @@ CREATE TABLE IF NOT EXISTS hyperlane_transfers (
     transfer_status         TEXT NOT NULL,
     transfer_status_message TEXT,
 
+    transfer_value TEXT,
+    max_gas_price_pct INTEGER,
```

Comment:
relay profitability options are now stored in the db along with the transfers info